jsonformatter.go

```diff
@@ -24,6 +24,7 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+// JSONFormatter will format logs in JSON with
```

Comment:
with what? 🤔 


Maybe better?

```suggestion
// JSONFormatter struct formats logs in JSON following Mia-Platform guidelines.
```